README.md

```diff
@@ -73,6 +73,20 @@ puck --extra-files config/*.yml
 
 The more advanced `:extra_files` feature, where the destination paths within the JAR file is specified as described below, is not available as a command line option.
 
+If your application is depending on another Jar you can merge it into the application Jar using the `:merge_archives` option:
```

Comment:
Jara uses "JAR" in its readme, Puck uses "Jar". I usually complain about other people not being consistent, sigh.